WinFormsUI/MainForm.cs

```diff
@@ -17,18 +19,43 @@ private async void MainForm_Load(object sender, EventArgs e)
 		{
 			State.OnStateUpdated += RefreshList;
 			State.OnStateUpdated += RefreshChart;
+			State.OnStateUpdated += RefreshCategories;
 			OnFilteringUpdated += RefreshList;
 			OnFilteringUpdated += RefreshChart;
 
+			clbCategories.ItemCheck += async (o, e) => 
+			{ 
+				await Task.Delay(100);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=4966361)